terraform/modules/iam_role_policy/concourse_iaas_worker/policy.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
```

Comment:
I want to add a comment here to state this is intended to be a clone of arn:aws-us-gov:iam::aws:policy/AdministratorAccess so future people don't make the same mistake we did / wonder why we have such a broad policy in place, but no comments in json.

WDYT about a comment in the TF where this is referenced?